server/src/controllers/Events/resolver.ts

```diff
@@ -324,20 +416,37 @@ ${unsubscribe}
     });
 
     // TODO: Handle tags
+    const start_at = new Date(data.start_at) ?? event.start_at;
     const update: Prisma.eventsUpdateInput = {
       invite_only: data.invite_only ?? event.invite_only,
       name: data.name ?? event.name,
       description: data.description ?? event.description,
       url: data.url ?? event.url,
       streaming_url: data.streaming_url ?? event.streaming_url,
       venue_type: data.venue_type ?? event.venue_type,
-      start_at: new Date(data.start_at) ?? event.start_at,
+      start_at: start_at,
       ends_at: new Date(data.ends_at) ?? event.ends_at,
       capacity: data.capacity ?? event.capacity,
       image_url: data.image_url ?? event.image_url,
       venue: { connect: { id: data.venue_id } },
     };
 
+    if (update.start_at !== event.start_at) {
```

Comment:
This will be true if `data.start_at` exists, right?  Even if it's exactly same date as `event.start_at`.  It would be a bit safer to use [isEqual](https://date-fns.org/v2.28.0/docs/isEqual) to see if anything has changed.

Reply:
I believe with unchanged start_at date in form this was being evaluated to false. For clarity and more confidence I've changed it to use `isEqual` though.